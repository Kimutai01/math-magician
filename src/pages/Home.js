import React from 'react';
import styles from './Home.module.css';

function Home() {
  return (
    <div className={styles.home}>
      <h1>Welcome to our page New</h1>
      <p>
        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Rerum
        blanditiis eius neque cumque, similique placeat exercitationem, sunt hic
        laudantium mollitia cupiditate veritatis doloremque rem eligendi?
        Maiores nihil reiciendis, quisquam, aliquid cumque magni et a possimus
        dolorem architecto magnam, rem accusantium voluptatibus eligendi fuga
        illum est deleniti minima numquam pariatur iste? Architecto atque animi
        cum libero provident ipsa recusandae dicta dolorum laboriosam. Veniam
        animi cupiditate eius iste labore omnis dolore in? Asperiores maxime
        tempora, nulla quod deleniti aliquam, quae deserunt amet, id explicabo
      </p>
    </div>
  );
}

export default Home;
