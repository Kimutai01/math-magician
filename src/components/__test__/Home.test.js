import renderer from 'react-test-renderer';
import Home from '../../pages/Home';

test('Home component testing with snapshot', () => {
  const TREE = renderer.create(<Home />).toJSON;
  expect(TREE).toMatchSnapshot();
});
