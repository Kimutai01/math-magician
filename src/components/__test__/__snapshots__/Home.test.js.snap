// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component testing with snapshot 1`] = `[Function]`;
